resources/function_help/json/to_dm

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "to_dm",
+  "type": "function",
+  "description": "Convert a coordinate to degree, minute.",
+  "arguments": [
+    {"arg":"float","description":"A coordinate in EPSG:4326."},
+    {"arg":"axe","description":"The axe of the coordinate. Either 'x' or 'y'."},
+    {"arg":"precision", "description":"Number of decimals"},
+    {"arg":"formatting", "description":"Optional parameter about formatting. Either empty or 'aligned' or 'suffix'."}
```

Comment:
This should be

    {"arg":"formatting", "optional": true, "description":"Designates the formatting type. Acceptable values are NULL, 'aligned' or 'suffix'."}

Using `"optional"=true` means that the parameter will be visually formatted as an optional parameter.